did-git-spec.md

```diff
@@ -125,29 +125,36 @@ Example:
 
 ## Read
 
 Notes: since we don't know the repo did string when the repo DID document is created, the repo DID document will not contain the "id" did string for the repo. The `git did read` operation will "resolve" the "id" by looking up the SHA1 hash of the commit that added the DID document to the repo and dynamically add the "id" member to what is rendered to the user.
 
 ## Update 
-- repo.did
-    - updating service_endpoints?
-- [personal-hash].did
-    - is any signature/crypto verification required to update a [personal-hash].did, or, for example, can I change your keys and commit it to the repo to effectively lock you out?  Or is change to .did/[personal-hash].did only allowed where [personal-hash].did matches the signer?
+**1. Repository DID**
+
+There aren't currently any keys associated with the respository, so there will ne no need to update the Public Keys in the DID document.  The process to change the maintainers or the canonical endpoint should be defined in the `.did/governance.md` file.
+
+**2. Contributor DID**
+
+To update a Contributor DID document, remove the previous DID document from the repository and commit.  This commit should be signed by the private key of the DID document being removed.  Then create a new DID document with the appropriate information and commit to the repository.  This commit should be signed by the private key of the DID document being added.
+
+TODO:
+
+1. We need a way to verify that the person making each of these changes has the appropriate keys to prevent unauthorized updates.  Is there a cleaner way to do this?
 
 ## Delete (Revoke)
 
-In the context of a repository, deletion takes two forms.
-    - Deletion of the repository did itself
-    - Deletion of a specific did relative to the reposistory
+**1. Repository DID**
+
+The DID for the repository can be deleted by removing the `.did/repo.did` file from the respository and performaing a commit.  The signature of this commit should correspond to one of the maintainers in the `.did/repo.did#authrntication list. 
     
-### Deletion of individuals
+**2. Contributor DID**
 
-Deletion of a personal did will remove the file from ```.did/[personal-hash].did```
-    - the [personal-hash].did is maintained in history which supports provability...
-    - and git porcelain is used to facilitate easy access to the information and the formation of the union of all current and historical personal-dids.
-- Deletion of a repository did is achieved by deleting the .did/repo.did file
+Deletion of a personal did will remove the file from `.did/<author key id>.did`.  The `<author did key>.did` is maintained in history which supports provability.  The commit associated with this change should correspond to one of the maintainers in the `.did/repo.did#authrntication list or the private key of the associated DID document itself.
```

Comment:
I think it is clearer to change "The commit associated with this change..." to "The signature over this change..."

Reply:
And again, fix `did/repo.did#authorization`